addons/mail/static/src/models/notification_request_view.js

```diff
@@ -1,12 +1,29 @@
 /** @odoo-module **/
 
 import { registerModel } from '@mail/model/model_core';
-import { one } from '@mail/model/model_field';
+import { attr, one } from '@mail/model/model_field';
+
+import { sprintf } from '@web/core/utils/strings';
 
 registerModel({
     name: 'NotificationRequestView',
     identifyingFields: ['notificationListViewOwner'],
+    recordMethods: {
+        /**
+         * @private
+         * @returns {string}
+         */
+        _computeHeaderText() {
+            return sprintf(
+                this.env._t("%s has a request"),
```

Comment:
You may want to name the placeholder to provide some more context to the translators. I can think of some languages that have different rules depending on whether the subject is a human, an animal, an inanimate object… and for which this kind of information matters.

Reply:
Change done